build-tests/heft-webpack-basic-test/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "noUnusedLocals": true,
     "types": ["jest", "webpack-env"],
 
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
Note: This is a somewhat clumsy solution. In a separate PR, I will provide a way for `jest-build-transform` to point at the correct `lib` folder for CommonJS.